### DISCUSSION TITLE
Never combine taskbar icons

### DISCUSSION THREAD

**Discussion by Aymkdn** [valinet/ExplorerPatcher#2639] [Q&A]
Hello,

I use ExplorerPatcher on my personal computer and it works great. I wanted to install it on my professional computer, but we have a lot of restrictions and I cannot install it. I have some administrator rights, but it won't let me install a .exe program that is not signed by an approved vendor.

I'm only interested by **ungrouping the taskbar icons**. I know Windows 11 recently updated to provide this option, but at my work we're still using a 2-years old version (22000.2652 21H2)…

I tried to change the registry for `TaskbarGlomLevel` and `MMTaskbarGlomLevel` to put 2, but no effect. I probably need to do something else to make it work, but I cannot figure out what.

Do you have more information on how I can have the "Never combine" for the taskbar? Or if there is another way to install ExplorerPatch (without executing a EXE file, like copying DLL somewhere)?

I found this answer: https://superuser.com/a/1708359/556381 – I didn't try because it says it will disable the Start Button and Win+V (that I use a lot).

Thanks

**Comment by Aymkdn** [comment]
I found that I could do `ep_setup.exe /extract` to get a bunch of `.dll` files. I also found an old [README](https://github.com/valinet/ExplorerPatcher/blob/3ab52f251c3adc8dcc5ea5270635463a028a960c/README.md) that says about moving a `dxgi.dll` file in `C:\Windows`, but the extracted files are different now. I'm not sure if I should move all the `*.dll` files to `C:\Windows`?

**Reply by Aymkdn** [reply]
OK, I found a solution: 

1. I copied `C:\Windows\dxgi.dll` from my personal computer to the pro one
2. I copied `C:\Program Files\ExplorerPatcher` from my personal computer to the pro one

After restarting Windows Explorer, it seems to work 🎉

**Reply by pyrates999** [reply]
You need admin privileges and the ability run unsigned exe's in order to install updates for EP.  I would not be installing it on your work computer as they can take away your admin privileges at any time.
